packages/language-server/src/types.ts

```diff
@@ -1,10 +1,12 @@
 // These settings are defined in the package.json
 export type Neo4jSettings = {
+  connectionKey: string;
```

Comment:
I'm going to remove this property from the Neo4jSettings type since it's not needed

but the connection key is a randomly generated 16 character slug when the connection is first created, it's not generated by the user and we don't allow it being modified

it will be used as the key in the connection key/value pair storage, so in global storage we'll have

```
    ' connections': {
        'example-connection-key': {
            ...
        },
        'another-key': {
            ...
        }
    }
```